Library/Formula/datomic.rb

```diff
@@ -1,9 +1,9 @@
 class Datomic < Formula
   desc "Database separating transactions, storage and queries"
   homepage "http://www.datomic.com/"
-  url "https://my.datomic.com/downloads/free/0.9.5206"
-  version "0.9.5206"
-  sha256 "b5b406d301b3e3fd4ff61d3e20946fb67fc5adae269bc2ee3bbca0819e02de3c"
+  version "0.9.5302"
+  url "https://my.datomic.com/downloads/free/#{version}"
```

Comment:
No worries, the tone was intended as friendly, however badly tone comes across in text. Indeed, The Cask are fans of the inlined version, but we don't use it.

We're slowly moving towards proper API docs which will help. The Cookbook is a bit of a behemoth. If you wanted to file a PR though we'd certainly be happy to review it :)